.github/workflows/create-release.yaml

```diff
@@ -0,0 +1,91 @@
+name: Create release
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    secrets:
+      GH_PROJECT_ACCESS_TOKEN:
+        required: true
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set version
+        id: version
+        run: |
+          VERSION=$(echo ${{ github.ref }} | sed -e "s#refs/tags/v##g")
+          echo "version=${VERSION}" >> $GITHUB_OUTPUT
+
+      - name: Set up JDK 8 and 11 (default 8)
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: |
+            11
+            8
+
+      - name: Checkout release note scripts
+        uses: actions/checkout@v4
+        with:
+          repository: scalar-labs/actions
+          token: ${{ secrets.GH_PROJECT_ACCESS_TOKEN }}
+          path: ${{ github.workspace }}
+          sparse-checkout-cone-mode: false
+          sparse-checkout: |
+            release-note-script/src/main/java
+
+      - name: Move scripts to the working directory
+        run: cp ${{ github.workspace }}/release-note-script/src/main/java/* ${{ github.workspace }}
+
+      - name: Create release note body
+        id: rn_body
+        env:
+          GH_TOKEN: ${{ secrets.GH_PROJECT_ACCESS_TOKEN }}
+        run: |
+          $JAVA_HOME_11_X64/bin/java ReleaseNoteCreation.java ${{ github.repository_owner }} ScalarDB ${{ steps.version.outputs.version }} ${{ github.event.repository.name }} > rnbody.md
+          cat rnbody.md # For debug
+
+      - name: Create release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_PROJECT_ACCESS_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: ${{ github.ref }}
+          # TODO: Remove '#' and enable "body_path: rnbody" when every PR becomes including the release note section.
+          # Until that, the release note body is created by actions/create-release automatically.
+          # body_path: rnbody.md
```

Comment:
Even before every PR becomes including the release note section, I think we can include `body_path` for testing since we can overwrite the generated body. What do you think?

Reply:
I see. My concern is that if there are some PRs that don't have the `Release Notes` section in the PR description, it might not be included generated release note body. And in that case, we have to find such PRs. 

If we can find them without too much effort, I agree with you to remove the comment and place the generated release note body to the release note as a test.

Maybe my concern is a minor. I hope to hear your opinion about that.

Reply:
Ah, either way, as a test, we should check whether all the PRs that should be included in the release notes are included in the generated release notes themselves.

Sorry for the lack of this perspective. I'll remove those comments and include `body_path` for testing.

Please ignore my previous comment🙇‍♂️

Reply:
Done at 1c6536f9